src/main/java/de/muenchen/isi/domain/service/AbfrageService.java

```diff
@@ -205,4 +256,44 @@ protected void throwAbfrageStatusNotAllowedExceptionWhenStatusAbfrageIsInvalid(
             throw new AbfrageStatusNotAllowedException(message);
         }
     }
+
+    /**
+     * Überprüft ob das Bauvorhaben wo die Abfrage zugeordnet ist keine andere Abfrage hat welche eine Relevante Abfragevariante hat.
+     *
+     * @param abfrage wo man eine Relevante Abfragevariante setzten möchte
+     * @throws UniqueViolationException          falls schon eine Relevante Abfragevariante exisitiert
+     * @throws BauvorhabenNotReferencedException falls die Abfrage keinem Bauvorhaben zugeordnet ist
+     */
+
+    private void checkForUniqueRelevantAbfragevariante(final InfrastrukturabfrageModel abfrage)
```

Comment:
In Methodenname beschreiben was diese macht.

Bei einem Check würde ich ein boolean als Rückgabewert erwarten.

Eher `throwExceptionIfNoBauvorhabenIsReferencedOrAnAbfragevarianteIsRelevant` oder ähnlich.